docs/engineering/dev-process.md

```diff
@@ -0,0 +1,56 @@
+---
+id: dev-process
+title: Development Process
+sidebar_label: Development Process
+---
+
+
+
+## Releasing Code
+0. A release owner is delegated to follow the process below and work with the
+   team to tie up any loose ends
+0. The pre-flight checklist:
+    0. Ensure there are no critical patches for this tag that haven't landed
+       yet
+    0. Fill out a new section in the [deployment doc][deployment-doc] for the Train you are tagging
+    0. Ensure you've got appropriate qa signoffs (**todo: link to doc**)
+    0. Ensure you don't have any modified files or code laying around before
+       you start the tag (**todo: does it matter what your github origins
```

Comment:
@vbudhram do you know this answer?

Reply:
No, it doesn't. The release script will automatically create the private ones for you, https://github.com/mozilla/fxa/blob/master/release.sh#L324.